pkgs/development/python-modules/pysonos/default.nix

```diff
@@ -26,17 +26,17 @@ buildPythonPackage rec {
     sha256 = "0a0c7jwv39nbvpdcx32sd8kjmj4nyrd7k0yxhpmxdnx4zr4vvzqg";
   };
 
-  propagatedBuildInputs = [ xmltodict requests ifaddr ];
+  propagatedBuildInputs = [ ifaddr requests xmltodict ];
 
   checkInputs = [
-    pytest pylint flake8 graphviz
-    mock sphinx sphinx_rtd_theme
+    pytestCheckHook
+    mock
     requests-mock
   ];
 
-  checkPhase = ''
-    pytest --deselect=tests/test_discovery.py::TestDiscover::test_discover
-  '';
+  disabledTests = [
+    "test_desc_from_uri" # disabled broken test
```

Comment:
That comment is not very meaningful, wouldn't you agree? I'm looking for an explanation like
- Issue is known upstream (`<link to issue>`)
- Requires network access

Reply:
I'm not sure what's interfering with the test here. There is no existing upstream issue. The failing test should have nothing to do with network access, as it seems to only test a URI parsing code there.

Reply:
Then its best to file an upstream issue and discuss with the maintainer.

Reply:
We get the default return value here, because nothing could be found. Something about an account, something about musicservice. This could be network related with graceful fallback.

https://github.com/amelchio/pysonos/blob/ec14789b339450cbd341d7f9dbbfa8a3af6df8ae/pysonos/music_services/music_service.py#L870